spec/support/vcr.rb

```diff
@@ -0,0 +1,16 @@
+VCR.configure do |config|
```

Comment:
I know this may be the case of personal preference, but I find VCR incredibly hard to maintain. 1) If someone needs to make changes in the future and 2) to see in the tests what the actual expected API call looks like. 

I'll throw it here that it could be easier to use `webmock` and nicely stub the expected calls, errors, etc. directly in the spec. What do you think?

Reply:
Am happy to go with it for now.

Reply:
I think there's a lot of noise with the various vcr cassettes which I don't really understand are they for. But the main reason is that you can't really test service errors, if the api is unavailable, or returns 400 error, etc. And we need to have those covered. Or is there a way to do this?

Reply:
Well, it does make an actual call. If we are stubbing the responses, then we are not really testing if the service is unavailable, just how we respond to that scenario. We can combine webmocks for those scenarios, no worries